src/utils/bytesParsers/formatNumber.ts

```diff
@@ -0,0 +1,107 @@
+import {formatNumber, roundToPrecision} from '../dataFormatters/dataFormatters';
```

Comment:
I suppose formatNumber shouldn't be in bytesParsers as it has small relevance to bytes operations logics